lib/utils/express/src/test/test.js

```diff
@@ -11,6 +11,8 @@ const express = require('..');
 /* jshint undef: false */
 /* jshint unused: false */
 
+debug.enable('abacus-express');
```

Comment:
As far as I can see we enable debugging for just a single test. Can we have it wrapped in context and set/unset with before/after?